pkg/daemon/ceph/osd/device.go

```diff
@@ -483,13 +480,12 @@ func WriteConfigFile(context *clusterd.Context, cluster *cephconfig.ClusterInfo,
 	if err != nil {
 		return err
 	}
-	confFile := getOSDConfFilePath(cfg.rootPath, cluster.Name)
-	util.WriteFileToLog(logger, confFile)
```

Comment:
is the config written to the log elsewhere? it's nice to see the config in the log for each of the daemon init containers

Reply:
It's logged at the end of `GenerateConfigFile` now so it's logged automatically.